src/ngraph/frontend/onnx_import/onnx.cpp

```diff
@@ -99,6 +99,11 @@ namespace ngraph
             OperatorsBridge::register_operator(name, version, domain, std::move(fn));
         }
 
+        OperatorSet get_operator_set(std::int64_t version, const std::string& domain)
```

Comment:
This is not what you need. Instead you supposed to provide a function e.g. `std::set<std::string> get_operators_list(version, domain)`.  You need the names of operators available (supported) for the given domain and version, and not the internals of ONNX importer exposed.